store/src/java/com/zimbra/cs/account/AccountServiceException.java

```diff
@@ -417,4 +419,13 @@ public static AccountServiceException TWO_FACTOR_AUTH_REQUIRED() {
     public static AccountServiceException ALIAS_EXISTS(String name) {
         return new AccountServiceException("email address alias already exists: "+name, ALIAS_EXISTS, SENDERS_FAULT, null);
     }
+
+    public static AccountServiceException NEED_CAPTCHA() {
+        return new AccountServiceException("show Captcha", NEED_CAPTCHA, SENDERS_FAULT, null);
```

Comment:
nit, use "CAPTCHA" (or less desirably "captcha") but not "Captcha" - all caps seems to be preferred ex: https://en.wikipedia.org/wiki/CAPTCHA or https://www.google.com/recaptcha/